boards/DIM/Inc/App/App_SevenSegDisplay.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <stdbool.h>
+#include "App_SharedConstants.h"
+
+struct SevenSegHexDigit
+{
+    // If enabled is false, then value is ignored.
+    bool          enabled;
+    enum HexDigit value;
+};
+
+struct SevenSegDisplay *
+    App_SevenSegDisplay_Create(void (*set_hex_digit)(struct SevenSegHexDigit));
+
+void App_SevenSegDisplay_Destroy(struct SevenSegDisplay *seven_seg_display);
+
+void App_SevenSegDisplay_SetHexDigit(
```

Comment:
Renamed `SetValue` and the like to be `SetHexDigit` and `SetHexDigits` because I think it makes it much more obvious what this function is doing and what input it expects (i.e. It only writes hex values and not any ascii value)